prometheus/histogram.go

```diff
@@ -655,6 +670,48 @@ func MustNewConstHistogram(
 	return m
 }
 
+func NewConstHistogramWithExemplar(
```

Comment:
Can we use `NewConstHistogramWithExemplar` in `NewConstHistogram` for less code repetition?

Reply:
Just to clarify, did you mean to use `NewConstHistogram` inside `NewConstHistogramWithExemplar`? My goal was to keep the existing API unchanged but whatever you folks prefer is fine by me.

Reply:
no, opposite. I think you misunderstood, let me show in PR to your PR 🙃